cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,14 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io/#login');
 
+    cy.get('input[type="email"]').type('ruviw@mailinator.com');
```

Comment:
Hardcoding credentials within the test script is not a good practice. You should consider using environment variables to store sensitive information like email and password.